src/views/List.jsx

```diff
@@ -1,15 +1,39 @@
 import { ListItem } from '../components';
+import { useState } from 'react';
 
 export function List({ data }) {
-	const listItemsToDisplay = data.map((item) => {
+	const [searchInput, setSearchInput] = useState('');
+	const [displayList, setDisplayList] = useState(data);
+	const filterDisplay = (currentInput) => {
```

Comment:
**Suggestion**: Add a space between the state and the filter function for code readability.